### ISSUE TITLE
"Undo" button for annotations

### ISSUE THREAD

**Issue by ChiaraPalladino** [pelagios/recogito2#295]
This somehow relates to issue #74 and #149 : we don't have an "undo" button for errors and that's _very_ necessary.

**Comment by rsimon** [comment]
Good point. I was thinking about this already and it's (remarkably) tricky. I still need to figure out how to best address this.

**Comment by ChiaraPalladino** [comment]
I learned it the hard way - by involuntarily marking all instances of accented alpha.

**Comment by rsimon** [comment]
ouch. well in that case a temporary workaround: go to Settings->Edit history. This would let you roll back a few edits. The history is unfinished insofar as it only shows the last 100 edits. So if you manage to break that limit with a single action, you might be out of luck (at the moment!), but usually it could help.

But undo/CTRL-Z is definitely a needed feature in any case, I fully agree.

**Comment by ChiaraPalladino** [comment]
luckily it was a very small document!
